### DISCUSSION TITLE
Recommanded Project for Case Study

### DISCUSSION THREAD

**Discussion by Django-Jiang** [mlc-ai/mlc-en#25] [General]
Hi, I am kind of new to the MLC topic, and when I finished ep4, I knew what I can do with tvm, but how are these techniques be applied in real model deployment process? I'd like to do some case study for better understanding, is there any open source project suggested I can study?

**Comment by yzh119** [comment]
You can use TVM for real model deployment, and you will see some deployment examples in episode 9 & 10.
